<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogInClass</class>
 <widget class="QMainWindow" name="LogInClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LeonCam</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/Resources/Images/logo.ico</normaloff>:/Resources/Images/logo.ico</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget
{
	/*background-color: rgb(174, 213, 255);*/
	background-image: url(:/Resources/Images/background.png);
}



</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="Lloading">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PBNewProfile">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>510</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#PBNewProfile{
color: rgb(255, 255, 255);
background-color:rgb(36, 118, 59)}
#PBNewProfile:hover
{
background-color: rgb(39, 129, 63);
}
/*border: none;
margin: 0px;
padding: 0px;*/</string>
    </property>
    <property name="text">
     <string>New profile</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEUsername">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEPassword">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>301</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CBKeepMeLoggedIn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#CBKeepMeLoggedIn{color: rgb(35, 167, 255);}
#CBKeepMeLoggedIn:hover
{
	
	color: rgb(89, 178, 255);
}</string>
    </property>
    <property name="text">
     <string>Keep me logged in</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PBLogIn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>301</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#PBLogIn{
background-color: rgb(0, 69, 104);
color: rgb(255, 255, 255);}
#PBLogIn:hover
{
background-color: rgb(0, 77, 116);
}</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="PBForgotPassword">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#PBForgotPassword{
color: rgb(255, 90, 75);
border: none;
margin: 0px;
padding: 0px;}
#PBForgotPassword:hover
{
color: rgb(255, 122, 82);
}</string>
    </property>
    <property name="text">
     <string>Forgot password?</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
